instrumentation/micronaut-1.0/build.gradle.kts

```diff
@@ -27,11 +27,8 @@ for (version in listOf(micronaut2Version)) {
         extendsFrom(configurations.testRuntimeClasspath.get())
```

Comment:
```diff
-        extendsFrom(configurations.testRuntimeClasspath.get())
+        extendsFrom(configurations.runtimeClasspath.get())
```
(the reason for this is we are specifying specific versions for this task, so we don't want to bring the versions used in the basic unit test task onto the classpath too)

Reply:
The additional test task should have the runtime classpath (not the test one though) to avoid repeating core deps (e.g. netty instrumentation etc.)

Reply:
My suggestion and your statement are the same :) currently, it's using the test runtime classpath, but we should change it to the core runtime classpath.

Reply:
changed to ` extendsFrom(configurations.runtimeClasspath.get())`